core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -129,16 +129,16 @@ public TableBuilder buildTable(TableIdentifier identifier, Schema schema) {
   }
 
   private Table loadMetadataTable(TableIdentifier identifier) {
-    String name = identifier.name();
-    MetadataTableType type = MetadataTableType.from(name);
+    String tableName = identifier.name();
+    MetadataTableType type = MetadataTableType.from(tableName);
```

Comment:
Why rename this?

Reply:
We use `name()` a few lines below now. I think it will be confusing to have `name()` that is the catalog name and `name` that is the metadata table name in the same method.

Reply:
Makes sense, but if this is the metadata table name (e.g. "files") instead of the table name, I think it might make more sense to be specific: `metadataName`?

Reply:
I feel like the current way is a bit more consistent with the overall structure of the method: the method name is `loadMetadataTable` and we use `baseTableIdentifier` to refer to the base table.